tasks/sep/mmz-002-key-handover/VALIDATION.md

```diff
@@ -0,0 +1,80 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of Key Handover process. This task will transfer ownership of the `ProxyAdminOwner` role to a different account. For mainnet, this is the 2-of-2 multisig of the Optimism Foundation and Security Council.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0xE75Cd021F520B160BF6b54D472Fa15e52aFe5aDD` (The current `ProxyAdmin` owner Safe for Mode, Metal, and Zora)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xE75Cd021F520B160BF6b54D472Fa15e52aFe5aDD)
+
+Enables the simulation by setting the threshold to 1:
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000004` <br/>
+  **Value:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+
+
+## State Changes
+
+**Notes:**
+- Check the provided links to ensure that the correct contract is described at the correct address.
+
+### `0xE17071F4C216Eb189437fbDBCc16Bb79c4efD9c2` (Zora's `ProxyAdmin`)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xE17071F4C216Eb189437fbDBCc16Bb79c4efD9c2)
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/0fb0dcbefc50882f1bb02fafcb27f47b463875c9/superchain/configs/sepolia/zora.toml#L43)
+
+State Changes:
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **Before:** `0x000000000000000000000000e75cd021f520b160bf6b54d472fa15e52afe5add` <br/>
+  **After:** `0x0000000000000000000000001eb2ffc903729a0f03966b917003800b145f56e2` <br/>
+  **Meaning:** The `ProxyAdmin` owner has been transfered to be the same as OP Sepolia. The correctness of this slot is attested to in the Optimism monorepo at [storageLayout/ProxyAdmin.json](https://github.com/ethereum-optimism/optimism/blob/e6ef3a900c42c8722e72c2e2314027f85d12ced5/packages/contracts-bedrock/snapshots/storageLayout/ProxyAdmin.json#L3-L8).
```

Comment:
A few things about this line: 
1. I'd add a link to the Superchain Registry showing that the after value is the OP ProxyAdminOwner
2. The storage layout link should use the earliest release tag that contains these storage layouts (ideally it'd be [bedrock](https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2fv1.0.0), but it doesn't have them).

Reply:
1. Makes sense, will add.
2. It looks like its available for the [ProxyAdmin storage layout](https://github.com/ethereum-optimism/optimism/blob/a12a09f7000ff785a4058998e09b366cbe5a3ed2/packages/contracts-bedrock/.storage-layout#L213). Can you explain why we'd use Bedrock instead of the current L1 contract version they're on?

Reply:
2. just because that's when the `ProxyAdmin` was last 'released' (even though I notice that it isn't listed in the release notes).